### PR TITLE
Fix error of ltt install and Add csi500 results to Transformer, ALSTM

### PR DESCRIPTION
**Pull request by lcrun**
Fix error of ltt install and Add csi500 results to Transformer, ALSTM

## Description
1. Fix error of ltt install when using run_all_model  
2. Add csi500 results to Transformer and ALSTM

## Motivation and Context
Apply csi500 to  Transformer and ALSTM

## How Has This Been Tested?

- [ ] Pass the test by running: `pytest qlib/tests/test_all_pipeline.py` under upper directory of `qlib`.
- [ ] If you are adding a new feature, test on your own test scripts.

## Screenshots of Test Results (if appropriate):
1. Pipeline test:
3. Your own tests:

## Types of changes

- [ ] Fix bugs
- [ ] Add new feature
- [x] Update documentation

### CODE REVIEW

File: examples/run_all_model.py [hunk 1]
```diff
@@ -325,10 +325,16 @@ def run(
                 execute(
                     f"{python_path} -m pip install light-the-torch", wait_when_err=wait_when_err
                 )  # for automatically installing torch according to the nvidia driver
-                execute(
-                    f"{env_path / 'bin' / 'ltt'} install --install-cmd '{python_path} -m pip install {{packages}}' -- -r {req_path}",
-                    wait_when_err=wait_when_err,
-                )
+                try:
+                    execute(
+                        f"{env_path / 'bin' / 'ltt'} install --install-cmd '{python_path} -m pip install {{packages}}' -- -r {req_path}",
```
[the remark this answers is not in the document]
Comment (jingedawang) [reply examples/run_all_model.py:330]: I also encountered the same error when running command
```
python run_all_model.py run 3 mlp Alpha158
```
Error message screenshot:
image

[the remark this answers is not in the document]
Comment (lcrun) [reply examples/run_all_model.py:330]: It looks like the "ltt" command not support "--install-cmd" option (Ubuntu 18.04.6 LTS) , so I add the extra logic to handle that when the user run model need torch with run_all_model.py,  for example  :
`python run_all_model.py run 3 alstm Alpha158 csi500`

The example error message: 
_Running CMD: /tmp/tmpeghauvco/bin/ltt install --install-cmd '/tmp/tmpeghauvco/bin/python -m pip install {packages}' -- -r /home/canl/qlib/examples/benchmarks/ALSTM/requirements.txt

Usage:
  ltt install [options] <requirement specifier> [package-index-options] ...
  ltt install [options] -r <requirements file> [package-index-options] ...
  ltt install [options] [-e] <vcs project url> ...
  ltt install [options] [-e] <local project path> ...
  ltt install [options] <archive url/path> ...

no such option: --install-cmd_

File: examples/benchmarks/README.md [hunk 2]
```diff
@@ -107,11 +107,15 @@ python run_all_model.py run 3 lightgbm Alpha158 csi500  # for models with random
 | Model Name | Dataset  | IC          | ICIR        | Rank IC     | Rank ICIR   | Annualized Return | Information Ratio | Max Drawdown |
 |------------|----------|-------------|-------------|-------------|-------------|-------------------|-------------------|--------------|
 | LightGBM   | Alpha158 | 0.0377±0.00 | 0.3860±0.00 | 0.0448±0.00 | 0.4675±0.00 | 0.1151±0.00       | 1.3884±0.00       | -0.0898±0.00 |
+| Transformer| Alpha158 | 0.0245±0.00 | 0.2183±0.04 | 0.0445±0.00 | 0.4324±0.04 | -0.0203±0.02      | -0.2960±0.32      | -0.2189±0.04 |
+| ALSTM      | Alpha158 | 0.0356±0.00 | 0.3351±0.00 | 0.0499±0.00 | 0.4958±0.00 |  0.0531±0.00      | 0.8070±0.00       | -0.1631±0.00 |
```
Comment (you-n-g) [inline examples/benchmarks/README.md:111]: Are you sure that the results of `ALSTM` of different runs nearly do not change? 
This does not  align with the previous results
image

Comment (you-n-g) [reply examples/benchmarks/README.md:111]: https://github.com/microsoft/qlib/tree/main/examples/benchmarks#alpha158-dataset

File: examples/run_all_model.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (you-n-g) [reply examples/run_all_model.py:330]: Thanks, I see now.
It looks like that this is a parameter supported by the older version of ltt.

**Comment by you-n-g** [comment]
Hi, did you encoutered any problems when solving this issue?

**Comment by lcrun** [comment]
Rerun the Alpha158 with ALSTM 20 times and updated the result to the README.md

**Pull request closed without merging**